packages/notebook/docker/requirements.txt

```diff
@@ -0,0 +1,23 @@
+anndata==0.11.1
```

Comment:
Any thoughts on `uv pip compile` over `requirements.txt`? The `uv pip compile` is traceable